app/dashboards/car_dashboard.rb

```diff
@@ -62,8 +62,11 @@ class CarDashboard < Administrate::BaseDashboard
 
   # Overwrite this method to customize how cars are displayed
   # across all pages of the admin dashboard.
-  #
   # def display_resource(car)
   #   "Car ##{car.id}"
   # end
+
+  def display_resource(car)
+    car.registration_number
```

Comment:
I think we should use a little more complex form for this label. Maybe "Brand Model: Registration number". For example: Honda Accord: WX 2345. If someone has many cars using only 'registration number' can be confusing.